pinot-core/src/main/java/org/apache/pinot/core/query/selection/SelectionOperatorUtils.java

```diff
@@ -339,6 +339,43 @@ public static DataTable getDataTableFromRows(Collection<Object[]> rows, DataSche
     return dataTableBuilder.build();
   }
 
+  /**
+   * The default value for each column type.
+   */
+  public static Serializable getDefaultValue(ColumnDataType dataType) {
```

Comment:
These formatters are already deprecated (used for the deprecated pql query response). I think you can put this as a private method in `GapFillGroupByDataTableReducer` because the default value might not be the same for different queries. You may refer to `DateSchema.ColumnDataType.convertAndFormat()` for the expected return type for each column data type

Reply:
Done